Doc/library/bdb.rst

```diff
@@ -73,6 +74,50 @@ The :mod:`bdb` module also defines two classes:
       Print the output of :meth:`bpformat` to the file *out*, or if it is
       ``None``, to standard output.
 
+   :class:`Breakpoint` instances have the following attributes:
+
+   .. attribute:: file
+
+      File name of the :class:`Breakpoint`.
+
+   .. attribute:: line
+
+      Line number of the :class:`Breakpoint` within
+      :attr:`file <bdb.Breakpoint.file>`.
+
+   .. attribute:: temporary
+
+      True if :class:`Breakpoint` at the (file, line) is temporary.
```

Comment:
I’m not familiar with this particular module, but it seems like these would read better if you dropped _the_, so they are just _at (file, line)_. Or do you need to move _the_ before _Breakpoint_?